preflight-packages/examples.jetstack.io/pods_basic/pods.rego

```diff
@@ -4,66 +4,66 @@ package pods
 import input["k8s/pods"] as pods
 
 # Rule 'container_cpu_limit'
```

Comment:
I think this comment needs to move down.

Reply:
Fixed in 3096205c404c72b6d325fc353c8e179599ccd7f4